CONTRIBUTING.md

```diff
@@ -1,75 +1,5 @@
- - PM verifies there is a corresponding JIRA ticket or GitHub issue.
-
-Next, the PM tests changes on a spinmint test server:
- - PM tests and verifies pull requests via the `Setup Test Server` label. Initial review is completed within 2 business days.
- - If changes are required, PM submits review as "Changes Requested", with a comment on the areas that require updates. Comment explains why changes are needed linking back to design principles.
-   - PM applies `Awaiting Submitter Action` label to more easily query the PR queue.
-   - Once changes are made, PM regenerates test server and repeats testing.
- - If bugs are found that are also on `master`, a new bug report is submitted in JIRA and linked to the PR. Bugs that are also found on `master` will typically not block merging of PRs.
- - If PR is approved, PM submits review as "Approved", commenting with areas that were tested.
- 
-#### Stage 2: Dev Review
-
-Two developers will review the pull request and either give feedback or approve the PR. Any comments should be addressed before the pull request moves on to the next stage.
-
- - PM reviewer adds `2: Dev Review` label, removes `1: PM Review` and `Setup Test Server` labels, and assigns 2 developers for review under Reviewers.
-   - At least one dev is assigned based on their [feature area](https://docs.mattermost.com/developer/core-developer-handbook.html#current-core-developers). Devs re-assign as needed.
- - Devs review the code and provide feedback, with initial review completed within 2 business days. Some areas to check include:
-   - Proper Unit Tests
-   - API documentation
-   - Localization
- - After the submitter has addressed and satisfied all reviewers' comments, `3: Ready to Merge` label is applied.
-
-#### Stage 3: Ready to Merge
-
-Review process is complete and the pull request is merged.
-
- - Dev assigns `3: Ready to Merge` label.
- - If Mattermost is not in release mode (between [major feature cut and release candidate cut](https://docs.mattermost.com/process/release-process.html)), the PR is merged into `master`.
- - If the PR is a major change, merge is postponed until the next release cycle.
-   - Dev calls out on the issue that it is a major change and it will be merged after branching.
-   - Once the current release is branched the PR can be merged into `master`.
-
-#### PR Merged
-
-After a PR is merged:
-- External Contributions: PM closes the [Help Wanted] issue and related JIRA ticket.
-- Internal Contributions: Core committer resolves the JIRA ticket.
-- PM follows up for docs and changelog, and QA for release tests.
+When you submit a pull request, it goes through a [code review process outlined here](https://developers.mattermost.com/contribute/getting-started/code-review/). We aim to start reviewing pull requests the week they are submitted, but the length of time to complete the process will vary depending on the pull request.
```

Comment:
nit: "The week they are submitted" implies we try to review things submitted on Friday the same day. Might suggest simplifying to the following, and letting the code-review documentation be authoritative:

```suggestion
When you submit a pull request, it goes through a [code review process outlined here](https://developers.mattermost.com/contribute/getting-started/code-review/). The length of time to complete the process will vary depending on the pull request.
```

Reply:
I would like to give contributors a rough idea when they can expect a review. What do you think about?

```suggestion
When you submit a pull request, it goes through a [code review process outlined here](https://developers.mattermost.com/contribute/getting-started/code-review/). We aim to start reviewing pull requests within a week, but the length of time to complete the process will vary depending on the pull request.
```

Reply:
Should we encourage a contributor to stay on this page and not read about our code review process? I'm wondering if we'd be better off leaving them without this information to force the click-through.

Reply:
I like that reasoning. But using this force, I would remove the information about the review length entirely.
```suggestion
When you submit a pull request, it goes through a [code review process outlined here](https://developers.mattermost.com/contribute/getting-started/code-review/).
```

Reply:
I'm in favour of that 👍